src/Packager.php

```diff
@@ -334,6 +392,110 @@ public function createZipArchive(): bool
         return $this->createZipArchive;
     }
 
+    /**
+     * Set meta description for SCORM package
+     *
+     * @param string|null $metadataDescription
+     */
+    public function setMetadataDescription(?string $metadataDescription): void
+    {
+        if (!$metadataDescription) {
+            $this->metadataDescription = "Build Date: " . date("m.d.Y") . "; Technology: html;";
+        } else {
+            $this->metadataDescription = $metadataDescription;
+        }
+    }
+
+    /**
+     * Get meta description for SCORM package
+     *
+     * @return string
+     */
+    public function getMetadataDescription(): string
+    {
+        return $this->metadataDescription;
+    }
+
+    /**
+     * Set metadata EntryIdentifier
+     *
+     * @param string $entryIdentifier
+     */
+    public function setEntryIdentifier(string $entryIdentifier)
+    {
+        $this->entryIdentifier = $entryIdentifier;
+    }
+
+    /**
+     * Get metadata EntryIdentifier
+     *
+     * @return string
+     */
+    public function getEntryIdentifier(): string
+    {
+        return $this->entryIdentifier;
```

Comment:
There will be a type exception if you won't have set this variable, but will have called this method. You should either use null coalescing operator here, or set this value in the constructor.